src/main/java/dev/emi/trinkets/mixin/PlayerManagerMixin.java

```diff
@@ -0,0 +1,25 @@
+package dev.emi.trinkets.mixin;
+
+
+import dev.emi.trinkets.data.EntitySlotLoader;
+import net.minecraft.network.ClientConnection;
+import net.minecraft.server.PlayerManager;
+import net.minecraft.server.network.ServerPlayerEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+/**
+ * Syncs slot data to player's client on login
+ *
+ * @author C4
+ */
+@Mixin(PlayerManager.class)
+public class PlayerManagerMixin {
+
+	@Inject(at = @At("TAIL"), method = "onPlayerConnect")
+	public void onPlayerConnect(ClientConnection connection, ServerPlayerEntity player, CallbackInfo cb) {
+      EntitySlotLoader.INSTANCE.sync(player);
```

Comment:
This is indented improperly, and the github is complaining about the lack of a trailing newline on this file